integration-tests/olminstall/runners/run_component_pytest.py

```
@@ -447,7 +416,7 @@ def _apply_non_blocking_timeout(comp: dict[str, str], ec: int, raw_ec: int) -> i
     if raw_ec != 124:
         return ec
     cluster_source = os.environ.get("CLUSTER_SOURCE", "").strip()
-    if cluster_source in ("", CLUSTER_SOURCE_EAAS) and comp.get("id") in ("ogx",):
+    if cluster_is_ephemeral_hosted_cluster_source(source) and comp.get("id") in ("ogx",):
```

Ruff (0.16.1)

[error] Undefined name `cluster_is_ephemeral_hosted_cluster_source`

(F821)